### PR TITLE
Implement a default API version for Kotlin

### PR DESCRIPTION
**Pull request by jkaster**
`LookerSDK` is the class name and `com.looker.sdk` is the namespace for API 4.0

`Looker31SDK` is the class name and `com.looker.sdk.api31` is the namespace for API 3.1

### CODE REVIEW

File: kotlin/src/test/KotlinExample.kt [hunk 1]
```diff
@@ -55,7 +54,7 @@ class KotlinExample {
         if(dash.dashboard_elements.isNullOrEmpty()) {
             return null
         }
-        val element = dash.dashboard_elements!!.filter { element -> (element.title?.toLowerCase() ?: "") == title }
```
Comment (rlm1023) [inline kotlin/src/test/KotlinExample.kt]: Hah, was this a bug?

File: kotlin/src/test/KotlinExample.kt [hunk 2]
```diff
@@ -100,11 +99,12 @@ class KotlinExample {
...
 
-fun main(args: List<String>) {
-    val x = KotlinExample()
-    val dash = x.findDashboardsByTitle("Test")
-    val tile = x.getDashboardTile(dash[0], "Test")
-    if (tile != null) {
-        val down = x.downloadTile(tile, "txt")
-    }
-}
+// TODO convert this to a functional test
```
Comment (rlm1023) [inline kotlin/src/test/KotlinExample.kt]: TODO

File: kotlin/src/test/TestMethods.kt [hunk 3]
```diff
@@ -202,6 +201,25 @@ class TestMethods {
             }
         }
     }
+
+    // TODO revive when last_viewed_at is available on the search endpoints
```
Comment (rlm1023) [inline kotlin/src/test/TestMethods.kt]: TODO

File: kotlin/src/test/KotlinExample.kt (hunk 1 above)
Comment (jkaster) [reply kotlin/src/test/KotlinExample.kt]: Probably. Got lucky with it. A compiler warning about the unused variable prompted me to fix it.

File: kotlin/src/test/KotlinExample.kt (hunk 2 above)
Comment (jkaster) [reply kotlin/src/test/KotlinExample.kt]: Yup. `TODO` is valid for this stage of the KotlinSDK

File: kotlin/src/test/TestMethods.kt (hunk 3 above)
Comment (jkaster) [reply kotlin/src/test/TestMethods.kt]: Also valid. Thanks for noticing!

**Pull request merged**